DiffractometerMockup.py

```diff
@@ -73,7 +73,82 @@ def init(self):
         self.image_width = 400
         self.image_height = 400
 
-        self.equipment_ready()
+        self.equipmentReady()
+        self.user_clicked_event = AsyncResult()
+
+        self.phiMotor = self.getDeviceByRole('phi')
+        self.sampleXMotor = self.getDeviceByRole('sampx')
+        self.sampleYMotor = self.getDeviceByRole('sampy')
+        self.camera_hwobj = self.getDeviceByRole('camera')
+        self.beam_info_hwobj = self.getObjectByRole('beam_info')
+        self.camera = self.camera_hwobj
+        self.beam_info = self.beam_info_hwobj
+       
+        self.sample_changer = self.getObjectByRole('samplechanger')
+        self.front_light_motor = self.getObjectByRole('frontlight')
+        self.front_light_swtich = self.getObjectByRole('frontlightswitch')
+        self.back_light_motor = self.getObjectByRole('backlight')
+        self.back_light_swtich = self.getObjectByRole('backlightswitch')
+
+        if self.phiMotor is not None:
```

Comment:
for example this could be like:

```
self.connect(self.motor_hwobj_dict['phi'],
                     "positionChanged",
                     self.phi_motor_moved)
```